conection_test.py

```diff
@@ -0,0 +1,84 @@
+#Document information
+document_data = {
+                 'author': 'checho651',
+                 'name': 'conection_test.py',
+                 'description': 'Short sctipt to test mongo_db_conection',
+                 'version': '2.0.0'
+                 }
+
+
+#Needed packages
+import pandas as pd
+import pymongo
+import time
+
+#Own files
+import mongo_db_conection as mdbc
+
+
+#Creates a pymongo.MongoClient in order to get access to de databases available.
+mongo_user = 'FundingBot'
+mongo_password = 'Funding3575'
```

Comment:
Need to be careful when uploading passwords to GitHub as they are publicly available